docs/scheduler/filter.md

```diff
@@ -48,6 +49,9 @@ $ swarm manage --filter=health --filter=dependency
 
 When creating a container or building an image, you use a `constraint` or
 `health` filter to select a subset of nodes to consider for scheduling.
+If there are nodes in the swarm that have a label named `containerslots`
+with a number-value, swarm will not launch more containers than given
```

Comment:
We refer label as key-value pair. 

> a label named `containerslots` with a number-value

=> a label with key as `containerslots` and a number-value